lib/stellar_core_commander/docker_process.rb

```diff
@@ -340,6 +352,9 @@ def launch_stellar_core fresh
       if @forcescp
         command += ["forcescp"]
       end
+      if fresh and @initial_catchup
```

Comment:
a small simplification: `if fresh` condition is already done on line 349. You could just add something like `command += if @initial_catchup then ["catchupat", "current"] else [] end`  after line 350 and it would be identical.

Reply:
I need this specific order of commands here, so it has to be last condition.